### ISSUE TITLE
[FEAT] Normalize confusion matrix by class support size

### ISSUE THREAD

**Issue by noamzbr** [deepchecks/deepchecks#1115]
**Is your feature request related to a problem? Please describe.**
Similarly to [sklearn](https://scikit-learn.org/stable/auto_examples/model_selection/plot_confusion_matrix.html), there should be an option to show the confusion matrix normalized by class support size, such that the quantity in each matrix cell is n_in_cell / number_of_samples_with_this_label. 
@shir22 IMO this option should be true by default, which is not the way it is in sklearn. 

**Describe the solution you'd like**
In case the flag is true, the display values will be the fractions, and the count (number of samples in cell) will be displayed in the hover values. In case the flag is false, the display value will be the count, the the fraction would be the hover value.

**Bonus points**
Also enable normalizing by the number of samples predicted to this class, rather than normalizing by the number of samples actually belonging to this class.

**Comment by shir22** [comment]
About the display - best to have both numbers not only in hover but also visible on matrix itself.
one of them (by which confusion matrix is colored) as a number, and the other (fraction/num of samples) in brackets
